scripts/build_both_dylib_and_so.sh

```diff
@@ -11,14 +11,14 @@ SERVER=gsa6.broadinstitute.org
 LIB_PATH="src/main/c"
 
 echo "building local files"
-./gradlew clean build
+./gradlew clean build || exit 1
```

Comment:
I don't understand why we need these `|| exit 1`s.  Isn't that what set -e does?